app/boards/shields/snap/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_SNAP_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "SNAP"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+	default y
+
+endif
+
+if SHIELD_SNAP_LEFT || SHIELD_SNAP_RIGHT
+
+config ZMK_SPLIT
+	default y
+ 
+if ZMK_DISPLAY
+
+config I2C
+	default y
+
+config SSD1306
+	default y
+
+config SSD1306_REVERSE_MODE
+	default y
+
+endif # ZMK_DISPLAY
+
+if LVGL
+
+config LVGL_HOR_RES_MAX
```

Comment:
The hot/very resolution settings can actually be removed, thanks to Zephyr changed with the Zephyr 3.0 update.